examples/tfengine/generated/team/additional_iam_members/terraform.tfvars

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+project_iam_members = {
+
+  0 = {
```

Comment:
see my comment in previous commits. We should still make these *_iam_members a list of objects. Having the index at the front is a bit weird.

Reply:
Github didn't sync your replies to the previous comment. Thanks for the examples!